gradle.properties

```diff
@@ -1 +1 @@
-org.gradle.jvmargs=-Xmx1536m
+org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=2048m -XX:+HeapDumpOnOutOfMemoryError
```

Comment:
IIRC the PermGen space was completely removed in JDK 8 and, thus, the `XX:MaxPermSize`, too: [Java 8: From PermGen to Metaspace](http://javaeesupportpatterns.blogspot.com/2013/02/java-8-from-permgen-to-metaspace.html)

Reply:
Hi @jritzerfeld,
thank you so much again :)